bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/bno/service/BNOConfigurationService.kt

```diff
@@ -17,6 +18,8 @@ class BNOConfigurationService(private val serviceHub : ServiceHub) : SingletonSe
         const val CACHE_REFRESH_PERIOD = "net.corda.businessnetworks.membership.cacheRefreshPeriod"
         // Specifies whether Notifications are enabled. Should be set to true if CACHE_REFRESH_PERIOD is not specified.
         const val NOTIFICATIONS_ENABLED = "net.corda.businessnetworks.membership.notificationsEnabled"
+        // Specifies the class for delegating BNO decisions to
+        const val BNO_DECISION_MAKER = "net.corda.businessnetworks.membership.bnoDecisionMaker"
```

Comment:
Great idea! How about renaming the property to something like "membershipAutoAcceptor". BNO decision maker is a bit vague, not clear what decisions it is making about.